tests/test_arp.py

```diff
@@ -22,6 +22,7 @@
 class TestSonicMIB(TestCase):
     @classmethod
     def setUpClass(cls):
+        tests.mock_tables.python_arptable.arp_filename = '/arp.txt'
```

Comment:
>arp_filename [](start = 42, length = 12)

Not used?

Reply:
This is used in mock python_arptable.
For single namespace, we will use arp.txt - this will have mgmt interfaces and front panel interfaces data.
For multiple namespace, we use host_arp.txt - this have only mgmt interface related data.
Rest of namespace information will be retrieved from mock appl_db.json.

Reply:
This is to test single ASIC, and we don't need to mock kernel ARP any more?

---
In reply to: [451193110](https://github.com/Azure/sonic-snmpagent/pull/129#discussion_r451193110) [](ancestors = 451193110)

Reply:
Yes, we don't need this, fixed.